.gitignore

```diff
@@ -1 +1,4 @@
 /resources
+
+.idea
```

Comment:
These should be in your global gitignore, not this repo's

https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore